crates/rpc/src/handlers/root.rs

```diff
@@ -0,0 +1,21 @@
+use ream_storage::db::ReamDB;
+use tree_hash::TreeHash;
+use warp::{
+    http::status::StatusCode,
+    reject::Rejection,
+    reply::{Reply, with_status},
+};
+
+use super::{BeaconResponse, state::get_state_from_id};
+use crate::types::id::ID;
+
+pub async fn get_root(state_id: ID, db: ReamDB) -> Result<impl Reply, Rejection> {
+    let state = get_state_from_id(state_id, &db).await?;
+
+    let state_root = state.tree_hash_root();
+
+    Ok(with_status(
+        BeaconResponse::json(state_root),
+        StatusCode::OK,
+    ))
+}
```

Comment:
can this function be put into the state.rs file and the function be called `get_state_root`